Bonsai.Configuration/PackageConfigurationUpdater.cs

```diff
@@ -69,6 +69,36 @@ string GetRelativePath(string path)
             return PathUtility.GetPathWithDirectorySeparator(relativeUri.ToString());
         }
 
+        static string CombinePath(string path1, string path2)
+        {
+            return PathUtility.GetPathWithForwardSlashes(Path.Combine(path1, path2));
+        }
+
+        static PackageConfiguration NormalizePathSeparators(PackageConfiguration configuration)
+        {
+            var normalized = new PackageConfiguration();
+            normalized.ConfigurationFile = configuration.ConfigurationFile;
```

Comment:
Even though it doesn't appear to actually matter, this should be normalized too because otherwise it looks like a bug at first glance.

Reply:
The `ConfigurationFile` property is not normalized since 1) it doesn't get persisted on file (`[XmlIgnore]`); and 2) it may refer to an absolute path, rather than a relative path. I am trying to avoid here the reverse Unix-problem where we might be unable to refer to specific paths on Windows because we normalize all backslashes (e.g. for a configuration file or bonsai environment loaded from a shared network drive folder).

My overall reasoning here is that it is always safe to apply path normalization if the path is a relative path, but not if it is an absolute path.

Reply:
Agreed, doing so would break UNC paths.